features/step_definitions/journey/registration_steps.rb

```diff
@@ -120,16 +120,16 @@
   @copy_cards = copy_cards
   @reg_balance = 154 + 5 * copy_cards
 
-  step("I want to register as an upper tier carrier")
-  step("I start a new registration journey in 'England' as a 'soleTrader'")
-  step("I complete my registration")
-  step("I pay by bank transfer")
+  @reg_number = SeedData.seed("outstanding_balance_and_cards.json", copy_cards: copy_cards)
 
-  @reg_number = @journey.confirmation_page.registration_number.text
-  puts "Registration " + @reg_number + " submitted with #{copy_cards} copy cards and outstanding balance"
+  puts "Registration " + @reg_number + " seeded with #{copy_cards} copy cards and outstanding balance"
 end
 
 Given("a limited company with companies house number {string} is registered as an upper tier waste carrier") do |ch_no|
+  # We don't want to use seeds here, as we are testing the ability of the journey
+  # to flag a registration for convictions
+  # Hence, make sure we use this scenario only once and use seeds instead
```

Comment:
These comments seem to contradict themselves. Can it be clearer whether we're using seeds or not here?

The same goes for the other comments in this file.

Reply:
Sorry, it meant "Do not reuse this scenario, instead create a new one and use seeds" as we don't want to go through the journey that generates convictions like this more than once. How can I make it clearer?

Reply:
Ah OK, thanks. I'd suggest:

```
# We recommend you don't reuse this scenario as it relies on creating a fresh registration with convictions, which will slow down tests.
# If you need registration data then use the seed functionality instead, as described in the README.
```